scrapers/covid-19/historical.js

```diff
@@ -77,6 +77,29 @@ const historicalV2 = async (keys, redis) => {
 			: parsedAtIndex['Province/State'].toLowerCase();
 		return newElement;
 	});
+	/* This is an edge case to handle recovered data for Canada.
+	   Since JHU dataset doesn't have recovered data for Canada
+	   disaggregated by Province/Region, the aggregate data is
+	   lost when recovered data is formatted in formatRecoveredData
+	   function. This  will create a non-existent Province/Region
+	   called recovered-aggregate with cases and deaths set to 0
+	   and recovered is set to aggregate recovered data. This data
+	   is pulled only if historical data for Canada is requested.
+	*/
+	for (let i = 0; i < result.length; i++) {
```

Comment:
Instead of looping over this big ass data object, can we do something in the loop above that picks it out of the group. Like before we `return`, we have some variable that we can set called `canadaData`.
```js
newElement.country = countryData.country || parsedAtIndex['Country/Region'];
newElement.countryInfo = countryData;
newElement.province = parsedAtIndex['Province/State'] === '' ? null
    : parsedAtIndex['Province/State'].toLowerCase();
// Like this
if (newElement.country.toLocaleLowerCase() === 'canada' && country['Province/State'] === '') `canadaData` = newElement
```

Reply:
Noted.